test/integration/generated/textanalytics/src/models/index.ts

```diff
@@ -525,51 +525,54 @@ export interface SentenceAssessment {
   isNegated: boolean;
 }
 
-export type AnalyzeBatchInput = JobDescriptor &
-  JobManifest & {
-    /** Contains a set of input documents to be analyzed by the service. */
-    analysisInput: MultiLanguageBatchInput;
-  };
+export interface AnalyzeBatchInput extends JobDescriptor, JobManifest {
+  /** Contains a set of input documents to be analyzed by the service. */
+  analysisInput: MultiLanguageBatchInput;
+}
 
-export type AnalyzeJobMetadata = JobMetadata & {
+export interface AnalyzeJobMetadata extends JobMetadata {
   displayName?: string;
-};
+}
 
-export type HealthcareJobState = JobMetadata &
-  Pagination & {
-    results?: HealthcareResult;
-    errors?: TextAnalyticsError[];
-  };
+export interface HealthcareJobState extends JobMetadata, Pagination {
+  results?: HealthcareResult;
+  errors?: TextAnalyticsError[];
+}
 
-export type AnalyzeJobState = AnalyzeJobMetadata &
-  TasksState &
-  Pagination & {
-    errors?: TextAnalyticsError[];
-    /** if includeStatistics=true was specified in the request this field will contain information about the request payload. */
-    statistics?: TextDocumentBatchStatistics;
-  };
+export interface AnalyzeJobState
+  extends AnalyzeJobMetadata,
+    TasksState,
+    Pagination {
+  errors?: TextAnalyticsError[];
+  /** if includeStatistics=true was specified in the request this field will contain information about the request payload. */
+  statistics?: TextDocumentBatchStatistics;
+}
 
-export type TasksStateTasksDetails = TaskState;
+export interface TasksStateTasksDetails extends TaskState {}
```

Comment:
perhaps generate an interface only if the list of properties is not empty.

Reply:
I think practically it is the same as having the alias. So I think I prefer the simplicity of the new code over handling the case to generate the alias.

The ideal thing may be to not generate it at all and give anything that has a `TasksStateTasksDetails` type  `TaskState` instead. But I think that work would be a lot more involved.

I'm fine with the new generated interface

Reply:
Thanks for your valuable perspective! I agree with all what you said :)